pkg/api/jira.go

```diff
@@ -0,0 +1,20 @@
+package api
+
+import (
+	apitype "github.com/openshift/sippy/pkg/apis/api"
+	"github.com/openshift/sippy/pkg/db"
+)
+
+func GetJIRAIncidentsFromDB(dbClient *db.DB) ([]apitype.CalendarEvent, error) {
+	// Get JIRA Incidents for display in calendar
+	var incidents []apitype.CalendarEvent
+	res := dbClient.DB.Table("jira_incidents").Select(`
+		start_time AS start,
+		resolution_time AS end,
+		key as jira,
+		key || ': ' || summary AS title,
+		'incident' AS phase,
+		'TRUE' as all_day`).Scan(&incidents)
```

Comment:
Any concerns about this growing over time?  I guess (hope) it would take some time for the incidents to get so big we wouldn't want them all in one query.  Didn't know if we could get the year or something from the calendar view.

Reply:
Makes sense, fixed.  The calendar sends the start/end ranges anyway for what the user is looking at.